nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/core/storage/sql/S3BinaryManager.java

```diff
@@ -455,6 +455,24 @@ public AmazonS3 getAmazonS3() {
         return amazonS3;
     }
 
+    /**
+     * Gets the bucket name.
+     *
+     * @since 11.1
+     */
+    public String getBucketName() {
+        return bucketName;
+    }
+
+    /**
+     * Gets the bucket name prefix.
+     *
+     * @since 11.1
+     */
+    public String getBucketNamePrefix() {
```

Comment:
Could you rename this to `getBucketPrefix` please? The internal field `bucketNamePrefix` is really misnamed but up to now it wasn't exposed externally so it wasn't that much of an issue. We can rename `bucketNamePrefix` to `bucketPrefix` in a second commit.